packages/bpk-component-image/src/BpkImage.js

```diff
@@ -27,10 +28,28 @@ import STYLES from './bpk-image.scss';
 
 const getClassName = cssModules(STYLES);
 
-const Image = props => {
+type BpkImageProps = {
+  altText: string,
+  height: number,
+  src: string,
+  width: number,
+  className: ?string,
+  inView: boolean,
+  loading: boolean,
+  onLoad: ?() => mixed,
+  style: ?{}, // eslint-disable-line react/forbid-prop-types
+};
+
+type ImageProps = {
+  altText: string,
+  hidden: boolean,
+  onImageLoad: Function,
```

Comment:
`Function` is the most relaxed form of function signature and should be avoided. Use something stricter like `() => mixed` instead